server/app/assets/javascripts/util.ts

```diff
@@ -47,3 +47,17 @@ export function assertNotNull<T>(
   }
   return value
 }
+
+// Throttles a function's execution.  Helpful for functions triggered by keystrokes.
+export function debounce<A = unknown>(
+  func: (...params: A[]) => void,
+  timeout = 200,
```

Comment:
Does 200ms feel laggy if you're typing fast?

Reply:
You're right, it does feel laggy.  I'll lower it.  Will try 100ms.

Reply:
Lowering it to 50 ms